scss/_type_matrix.scss

```diff
@@ -31,9 +37,13 @@
 ///
 /// @requires {mixin} oTypographySansSize
 ///
-@mixin oTypographySans($level) {
-	@include oTypographySansSize($level);
-	font-family: $o-typography-sans;
+@mixin oTypographySans($level, $load-progressively: false) {
```

Comment:
Sassdoc comments will also need the `requires {mixin} oTypographyProgressiveFont` too... (sorry)